### PR TITLE
added detect_without_unit for number range patterns having unit type …

### PR DESCRIPTION
**Pull request by ruthvik-17**
added detect_without_unit for number range patterns having unit type in between min-max range and min>max case

### CODE REVIEW

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 1]
```diff
@@ -36,15 +35,16 @@ def __init__(self, entity_name, language, data_directory_path, unit_type=None):
         self.tag = '__' + entity_name + '__'
         self.range_variants_map = {}
         self.unit_type = unit_type
-
+        self.language=language
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: E225 missing whitespace around operator

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 2]
```diff
@@ -210,12 +210,19 @@ def _get_number_range(self, min_part_match, max_part_match, full_match):
...
-            entity_unit = entity_dict[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]
+            if not entity_unit:
+                entity_unit = entity_dict[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]
 
         if self.unit_type and (
                 entity_unit is None or self.number_detector.get_unit_type(entity_unit) != self.unit_type):
             return number_range, original_text
 
+        if min_part_match and max_part_match:
+            if entity_value_min>entity_value_max:
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: E225 missing whitespace around operator

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

File: ner_v2/detectors/numeral/number/en/data/units.csv [hunk 3]
```diff
@@ -1,6 +1,10 @@
 unit_type,unit_value,unit_variants
 currency,rupees,rupees | rupee | rs | rupya | rupaya | rupaye | rupye | rupay | paisa | paise | inr | ₹
-currency,dollar,Dollar | usd | $
+currency,dollar,Dollar | dollars | usd | $
+currency,euro,Euro | euros | eur | 	€
+currency,pound sterling,Pound sterling | pound sterlings | quid | pounds | sterling | pound | gbp | £
+currency,cent,Cents | cent | c | ¢
+currency,pence,Pence | p
```
Comment (amansrivastava17) [inline ner_v2/detectors/numeral/number/en/data/units.csv]: avoid single char

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**